### ISSUE TITLE
group.delete() fails

### ISSUE THREAD

**Issue by KlausGlueckert** [andrewvillazon/pbipy#12]
delete() method seems not to be implemented

```
TypeError                                 Traceback (most recent call last)
Cell In[33], [line 3](vscode-notebook-cell:?execution_count=33&line=3)
      [1](vscode-notebook-cell:?execution_count=33&line=1) groups=pbi.client.groups(filter="name eq 'customer3'")
      [2](vscode-notebook-cell:?execution_count=33&line=2) group=groups[0]
----> [3](vscode-notebook-cell:?execution_count=33&line=3) group.delete()

TypeError: RequestsMixin.delete() missing 2 required positional arguments: 'resource' and 'session'
```

**Comment by andrewvillazon** [comment]
Try using the `PowerBI.delete_group()` method on the Power BI client, something like `pbi.delete_group(groups[0])` should do it. Let me know if that works.

I think this is a side effect of providing customized requests functionality through a Mixin. Will have a think about how to address.

**Comment by andrewvillazon** [comment]
See mentioned workaround. To be resolved in #13. Closing.

**Comment by andrewvillazon** [comment]
Now addressed in [v2.8.2](https://github.com/andrewvillazon/pbipy/releases/tag/v2.8.2)
